MiniApp/Classes/core/Display/MiniAppAdDisplayer.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+
+/// This class is the parent class meant to be overriden when implementing ads SDK
+/// - Tag: MiniAppAdDisplayer
+internal class MiniAppAdDisplayer: NSObject, MiniAppAdDisplayDelegate {
+    #if RMA_SDK_ADMOB
+    static let shared = AdMobDisplayer()
```

Comment:
I'm not sure that having a shared `MiniAppAdDisplayer` will work the way that we need.

The host app needs to declare the `MiniAppAdDisplayer` for each individual mini app. This is because some mini apps will use Ad Mob and some will use something else. So by default, a mini app should be launched without support for Ads, and it only supports Ads if a `MiniAppAdDisplayer` is supplied to `MiniApp.create`.